src/locales/en.json

```diff
@@ -169,16 +174,18 @@
       "whats_this2": "This calculator lets you estimate COVID risk and find effective safety measures for customizable situations. Examples: how risky is <1>a trip to my grocery store</1>? What's the safest way to <3>see a friend</3>? How much would it help to <5>wear a better mask at my workplace</5>?",
       "whats_this": "We’ve constructed a calculator that lets you estimate the risk of getting COVID from an activity or relationship in your daily life, using the <1>best research available</1>. We hope this tool will help hone your intuition, lower your stress levels, and figure out good harm-reduction strategies.</p><p>The first step in protecting your family and community is to avoid getting COVID yourself.",
       "see_video": "If you'd like to watch a video walkthrough of how to use the calculator, click <1>here</1>.",
-      "call_to_action": "Calculate the approximate COVID risk of an activity or relationship"
+      "call_to_action": "Calculate approximate COVID risk"
```

Comment:
COVID risk -> risk of contracting COVID?

Reply:
Done